.github/workflows/deliverables-push-main.yml

```diff
@@ -47,3 +47,5 @@ jobs:
       coordinator_port_http: 80
       network: regtest
       tag: main
+      oracle_endpoint: http://api.test.10101.finance:8081
```

Comment:
🙃 I know it's subjective, but could we consider some other port number, e.g. `8090` for the oracle? `8081` sounds like a temporary second version during upgrade of whatever we're normally running on port `8080`